### PR TITLE
Fix  a typo

### PR DESCRIPTION
**Pull request by thefrieddan1**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

**No mandatory dependencies were added on this pull request.**

### CODE REVIEW

File: Packs/Palo_Alto_Networks_WildFire/Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.py [hunk 1]
```diff
@@ -1562,7 +1562,8 @@ def main():  # pragma: no cover
         if len(TOKEN) > 32 and API_KEY_SOURCE not in ['pcc', 'prismaaccessapi', 'xsoartim']:
             # the token is longer than 32 so either PPC or Prismaaccessapi needs to be set
             raise DemistoException(
-                'API Key longer than 32 chars, agent value must be selected in the intergration instance.')
+                'API Key is longer than 32 characters,\
```
Comment (thefrieddan1) [inline Packs/Palo_Alto_Networks_WildFire/Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.py:1565]: Hi @ShirleyDenkberg ,
Please review my phrasing here.

Comment (ShirleyDenkberg) [inline Packs/Palo_Alto_Networks_WildFire/Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.py:1565]:
```suggestion
                'API Key is longer than 32 characters.\
```

File: Packs/Palo_Alto_Networks_WildFire/Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.py [hunk 2]
```diff
@@ -1562,7 +1562,8 @@ def main():  # pragma: no cover
...
+please select an "API Key Type" in the integration\'s instance configuration.')
```
Comment (ShirleyDenkberg) [inline Packs/Palo_Alto_Networks_WildFire/Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.py:1566]:
```suggestion
Select an "API Key Type" in the integration\'s instance configuration.')
```

File: Packs/Palo_Alto_Networks_WildFire/ReleaseNotes/2_1_36.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Palo Alto Networks WildFire v2
+
+- Fixed a typo in integration's configuration message.
```
Comment (ShirleyDenkberg) [inline Packs/Palo_Alto_Networks_WildFire/ReleaseNotes/2_1_36.md:6]:
```suggestion
- Fixed a typo in the integration's configuration message.
```

**Comment by ShirleyDenkberg** [comment]
@thefrieddan1 Doc review completed.

**Review by daryakoval — APPROVED** [review]

**Pull request merged**